### DISCUSSION TITLE
quick/strict-router

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#35] [General]
# quick/strict-router

为了简化路由注册方式，避免一些繁琐的参数处理细节，

https://goframe.org/quick/strict-router

**Comment by dabu680** [comment]
g.Meta `path:"/" method:"get"`这个写法是不是写错了

**Comment by 27933** [comment]
http://127.0.0.1:8000/hello?name=xxx&age=18

执行结果哪里应该是少个路由吧?

**Reply by 27933** [reply]
​​奇怪 我起的路由是这样的
  ADDRESS | METHOD | ROUTE  |      HANDLER      | MIDDLEWARE
----------|--------|--------|-------------------|-------------
  :8000   | GET    | /hello | main.(*Hello).Say |
----------|--------|--------|-------------------|-------------

**Reply by 27933** [reply]
奥奥 明白了 我在请求结构体中定义的路径不一样 所以不同

**Comment by tonyho** [comment]
为什么这里面不需要指定Say方法是Path / 的回调函数也会被自动调用？
